code/modules/clothing/masks/gasmask.dm

```diff
@@ -397,6 +397,67 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 		to_chat(user, span_notice("Your Mime Mask has now morphed into [choice]!"))
 		return TRUE
 
+/**
+ * # Stage Mask
+ *
+ * Mime stage mask
+ *
+ * A subtype of the regular mime mask with style choices
+ *
+ */
+
+/obj/item/clothing/mask/gas/mime/utopia
+	name = "stage mask"
+	desc = "A classic stage mask used by the masters of pantomime."
+	clothing_flags = MASKINTERNALS
+	base_icon_state = "utopia"
+	icon_state = "utopia_tragedy"
+	inhand_icon_state = null
+	w_class = WEIGHT_CLASS_SMALL
+	flags_cover = MASKCOVERSEYES
+	resistance_flags = FLAMMABLE
+	actions_types = list(/datum/action/item_action/adjust_style)
+	species_exception = list(/datum/species/golem)
+	fishing_modifier = 0
+
+/obj/item/clothing/mask/gas/mime/utopia/Initialize(mapload)
+	. = ..()
+	/// List of icons for our mask style
+	mimemask_designs = list(
+		"Tragedy" = image(icon = src.icon, icon_state = "utopia_tragedy"),
+		"Comedy" = image(icon = src.icon, icon_state = "utopia_comedy"),
+		"Anger" = image(icon = src.icon, icon_state = "utopia_anger"),
+		"Cluelessness" = image(icon = src.icon, icon_state = "utopia_clueless"),
+		"Emotionlessness" = image(icon = src.icon, icon_state = "utopia_emotionless"),
+	)
+	// society
+	if(prob(50))
+		icon_state = "utopia_comedy"
+
+/obj/item/clothing/mask/gas/mime/utopia/ui_action_click(mob/user)
+	if(!istype(user) || user.incapacitated)
+		return
+
+	var/list/options = list()
+	options["Tragedy"] = "[base_icon_state]_tragedy"
+	options["Comedy"] = "[base_icon_state]_comedy"
+	options["Anger"] = "[base_icon_state]_anger"
+	options["Cluelessness"] = "[base_icon_state]_clueless"
+	options["Emotionlessness"] = "[base_icon_state]_emotionless"
+
+	var/choice = show_radial_menu(user, src, mimemask_designs, custom_check = FALSE, radius = 36, require_near = TRUE)
+	if(!choice)
+		return FALSE
+
+	if(user.can_perform_action(src, ALLOW_RESTING))
+		icon_state = options[choice]
+		user.update_worn_mask()
+		update_item_action_buttons()
+
+		balloon_alert(user, "morphed into [choice]")
+		to_chat(user, span_notice("Your Stage Mask has now morphed into [choice]!"))
```

Comment:
why does this need to be in the chat box? You can literally see the change haha.

Imo the balloon alert should REPLACE the tochat